rotkehlchen/db/dbhandler.py

```diff
@@ -3319,17 +3319,39 @@ def should_save_balances(self, cursor: 'DBCursor') -> bool:
         now = ts_now()
         return now - last_save > period
 
-    def get_web3_nodes(self, only_active: bool = False) -> List[WeightedNode]:
+    def get_web3_node_by_id(self, identifier: int) -> WeightedNode:
```

Comment:
Can you put a docstring explaining what this is supposed to do and what it returns?

Also specify which errors this can raise. For far I see the `InputError`. The DB entries we always assume to be correct since they should be checked at addition

Reply:
done :+1: